pkg/controller/controller.go

```diff
@@ -422,3 +478,37 @@ func (ctrl *resizeController) markPVCAsFSResizeRequired(pvc *v1.PersistentVolume
 
 	return nil
 }
+
+func parsePod(obj interface{}) *v1.Pod {
+	if obj == nil {
+		return nil
+	}
+	pod, ok := obj.(*v1.Pod)
+	if !ok {
+		tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
```

Comment:
Can you pick a different name than "tombstone"?

Reply:
renamed to `staleObj`